### DISCUSSION TITLE
Why change src camera still look at the same point?

### DISCUSSION THREAD

**Discussion by funylion** [google/model-viewer#2298] [General]
Loading by src it look like this at first time.
![图片](https://user-images.githubusercontent.com/19399348/114986338-76df7000-9ec6-11eb-9410-a8a11f50e260.png)
And after ,I make a little change,
![图片](https://user-images.githubusercontent.com/19399348/114986456-9d9da680-9ec6-11eb-8aee-e98df7bfbf18.png)
Then change src to load the new one ,and it look like camera still look at the same point.
![图片](https://user-images.githubusercontent.com/19399348/114986556-b7d78480-9ec6-11eb-9bb9-0028bebdcd48.png)
It doesn't look like a bug, but is there a good solution?

**Comment by LaloYanez** [comment]
You will need to code your desired values inside a function that listens for the "load" event.

The camera attributes you will be setting a value for would be fieldOfView, cameraTarget and/or cameraOrbit. You can set your own or just use auto.

**Reply by funylion** [reply]
![图片](https://user-images.githubusercontent.com/19399348/115098941-70540580-9f65-11eb-8d63-ffa931c9f2de.png)
`const modelViewer = document.querySelector('#mymodel');
modelViewer.setAttribute('cameraTarget', `auto auto auto`);
modelViewer.setAttribute('cameraOrbit', `0deg 75deg 105%`);
modelViewer.setAttribute('field-of-view', `auto`);
`


Yes, I know these attributes, but I get the same result after modifying them. That's why I'm confused.

**Reply by LaloYanez** [reply]
The way you have them set only works when the script first loads. In order for it to work every time you swap models you have to include them when the "load" event happens.

```
modelViewer.addEventListener("load", function() {
   modelViewer.cameraTarget = "auto auto auto";
   modelViewer.cameraOrbit = "0deg 75deg 105%";
   modelViewer.fieldOfView = "auto";
  }
});
```

**Reply by funylion** [reply]
You let me know what's wrong with my code.Thank you very much for your help！
